android/avf/vm_config.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "nixos",
```

Comment:
todo make all of this configurable with a module and also update fileSystems accordingly